src/UI/Art/SpriteArtist/UnitSpriteSet.java

```diff
@@ -162,12 +175,22 @@ public void drawUnitIcons(Graphics g, Commander[] COs, Unit u, int animIndex, in
   {
     int unitHeight = turnDone.getFrame(0).getHeight();
 
-    // Draw the unit's HP if it is below full health.
-    if( u.getHP() < 10 )
+    ArrayList<BufferedImage> unitIcons = new ArrayList<BufferedImage>();
+
+    // Draw the unit's HP if it is not at full health.
+    if( u.getHP() != u.model.maxHP )
     {
-      BufferedImage num = SpriteLibrary.getMapUnitHPSprites().getFrame(u.getHP());
-      g.drawImage(num, drawX, drawY + ((unitHeight) / 2), num.getWidth(), num.getHeight(),
-          null);
+      BufferedImage num;
+      if( u.getHP() > u.model.maxHP )
+      {
+        num = SpriteLibrary.getMapUnitHPSprites().getFrame(1); // Tens place.
+
+        // Ones place shares space with the activity icons below if HP > 10.
+        unitIcons.add( SpriteLibrary.getMapUnitHPSprites().getFrame((u.getHP()-u.model.maxHP)) );
+      }
+      else
+        num = SpriteLibrary.getMapUnitHPSprites().getFrame(u.getHP());
+      g.drawImage(num, drawX, drawY + ((unitHeight) / 2), num.getWidth(), num.getHeight(), null);
     }
```

Comment:
I think I'll reserve final judgement until I see both versions of this... but I feel like this is kind of a jank way to show it.

Reply:
By "both" versions, do you mean:
![Screenshot from 2020-06-16 19-37-19](https://user-images.githubusercontent.com/16439262/85888914-1bfe5000-b7b0-11ea-8ccf-1a9db3ecc7a7.png)

![Screenshot from 2020-06-16 19-36-03](https://user-images.githubusercontent.com/16439262/85888901-14d74200-b7b0-11ea-94da-41ff1b9d5830.png)

Reply:
Yes. I mentioned my conclusion in the summary comment.

Reply:
Per your comment below, you like the second version better than the first? Once I figured how how to make the base-10 display work, I switched over since it looks far more intuitive.

Reply:
Yessir, I do like having it as a CO mark.
Having HP over 10 is non-standard behavior, and should be treated as a CO buff (because it is one).
Likewise, there *shouldn't* (pending possible scope creep, I guess?) be any unit sets that modify the max HP to be above 10.

I would also contest your claim about it being intuitive: IMO having any number in the bottom left of the unit indicates that the unit is injured. A 1 indicates its almost dead. Changing that expectation is unintuitive.

Reply:
> IMO having any number in the bottom left of the unit indicates that the unit is injured. A 1 indicates its almost dead. Changing that expectation is unintuitive.

... I think that the presence of a second digit would make things extremely clear to anyone who saw it. The unit markngs on the other are a whole new affordance we've added, in a spot that's only really used for bookkeeping unit metadata. Putting an HP value there in a different color will just create questions when people first see it.
I mean, folks will figure out what it all means either way after seeing it in action, but base-10 numbers seem more immediately clear.
I'd be open to more opinions though; nothing like actual end-user input to shape expectations.

Reply:
> ... I think that the presence of a second digit would make things extremely clear to anyone who saw it.

I guess? It's justified wrong, and can take the same space as other icons, though.

> The unit markngs on the other are a whole new affordance we've added, in a spot that's only really used for bookkeeping unit metadata. Putting an HP value there in a different color will just create questions when people first see it.

As it should, IMO. HP > 10 is *allowed*, but under no circumstances should it be considered *standard*.

> I mean, folks will figure out what it all means either way after seeing it in action, but base-10 numbers seem more immediately clear.
> I'd be open to more opinions though; nothing like actual end-user input to shape expectations.

How about you post in the test lab and/or AWBW's customization chat and see what peeps think? :stuck_out_tongue:

Reply:
> It's justified wrong

It's fixed-width.

> can take the same space as other icons

They rotate. That's an issue with putting it in the mark spot too, anyway.

> As it should, IMO ... under no circumstances should it be considered standard

Disagree ... Agree. Just because something is new doesn't mean it can't also make immediate sense.

> How about you post in the test lab and/or AWBW's customization chat

Could. They may just be annoyed that this isn't one of the "real" features though. Maybe I could placate them by promising to do productive work. Might be worth doing; I'll let your older brother weigh in on it in the meantime.

Reply:
> It's fixed-width.

*shrug*
> > can take the same space as other icons
> 
> They rotate. That's an issue with putting it in the mark spot too, anyway.

Not nearly as much as an issue, though, IMO.
For base-10 HP, you can see an APC or capping inf with "1" HP.
With the marker, you see an *unhurt" APC or capping inf, and you can wait around for other markers to appear if needed.

> Disagree ... Agree. Just because something is new doesn't mean it can't also make immediate sense.

Right, and I'd argue that "making immediate sense" is all about not violating the expectation of "HP number means fighting at less than 100% capacity"

> Could. They may just be annoyed that this isn't one of the "real" features though.

Maybe. I would be inclined to think they're more reasonable than that.

> Maybe I could placate them by promising to do productive work.

Though I can't imagine this would hurt too much, lol.

> I'll let your older brother weigh in on it in the meantime.

This implies he's meaningfully apprised of the situation?
~~and he's not your older brother?~~

Reply:
> Maybe. I would be inclined to think they're more reasonable than that

Yeah, I'm sure they would be.

> This implies he's meaningfully apprised of the situation?

Nope. Which is the point. I sent an email without explaining what either means. We'll see which one lets him figure it out.

> For base-10 HP, you can see an APC or capping inf with "1" HP.

On the one hand yes, this is possible (for about half a second or whatever). On the other hand, it's likely that other units will also be buffed (since this only happens via abilities, which are likely to impact many units), and the odds that an inf or APC is the only affected unit in sight is... vanishingly small. The odds that on top of that, the rotating icon will cause a user to make the wrong choice is even more vanishingly smallerer.
Even with the rotating icon (which affects both anyway), the time-to-understanding is almost certainly less with the base-10 numbers (not to mention that we could make some pretty absurd enemy boss units with this, or e.g. Omega Tech with +25 HP that doesn't go away).

Reply:
> Nope. Which is the point. I sent an email without explaining what either means. We'll see which one lets him figure it out.

A cunning strategy.

> On the one hand yes, this is possible (for about half a second or whatever). On the other hand, it's likely that other units will also be buffed (since this only happens via abilities, which are likely to impact many units), and the odds that an inf or APC is the only affected unit in sight is... vanishingly small.

Sure, but now you're requiring looking around at the other units... and even then, it's only 100% clear with more context about how the CO actually functions.
Say I send someone a screenshot and ask for an opinion. Now the person I'm asking either thinks the inf is 1 HP or thinks it's not capturing (maybe it shot one of my dudes last turn)

OTOH, the vast majority of the time, only one marker will be visible on a given unit at a time (since a CO with multiple markers is likely too complex and COs who apply them to others are fairly rare).
The *most* relevant information in most cases is that the unit is no less than 10 HP, and that's indicated unambiguously by leaving that corner of the unit blank.
You could run into, say, a stunned/revenge-marked unit at 11+ HP, and end up missing the more important piece of info in a screenshot, but that's only possible in a specific CO matchup vs happening in *any* match with Tech.

> this only happens via abilities

I would not make this assumption. I would design around this being a possible D2D (as has been proposed a few times)

> The odds that on top of that, the rotating icon will cause a user to make the wrong choice is even more vanishingly smallerer.

Fair, but we should do what we can to make the game state easily readable.

> Even with the rotating icon (which affects both anyway), the time-to-understanding is almost certainly less with the base-10 numbers

Fair, but IMO it's less readable for the player who knows what's going on (as detailed above)

> (not to mention that we could make some pretty absurd enemy boss units with this, or e.g. Omega Tech with +25 HP that doesn't go away).

IDK, I feel like that's a bridge too far from the basic AW contract.

> ~He totally is. I just wanted to distinguish him from your younger brother.~

~~Okay. Glad we got that one cleared up~~